### PR TITLE
#166741636 Feature to add a quiz component

### PR DESCRIPTION
**Pull request by MBenedicte**
#### What does this PR do?
Add a quiz component

#### Description of Task to be completed?
Having:
* Button component

#### How should this be manually tested?
After cloning the repo, run **npm start** command

#### Any background context you want to provide?
The project is implemented using react and redux

#### What are the relevant pivotal tracker stories?
#166741636

#### Screenshots (if appropriate)
Screen Shot 2019-06-18 at 10 58 58

### CODE REVIEW

File: src/assets/css/_opacity.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
-.black-opacity-1{background: rgba(0,0,0,0.2)}
+git .black-opacity-1{background: rgba(0,0,0,0.2)}
```
Comment (kagaramag) [inline src/assets/css/_opacity.scss]: Is this `git`?

File: src/components/Quiz/quiz.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+              className="card radius-3 bold large-text text-white center-align"
+            >
+              question
+            </div>
+            <div id="country" className="hide" />
+            <div className="row">
+              <div className="small-screen-3 medium-screen-3 large-screen-3">
+                <div className="input-field">
+                  <input className='medium-text large-screen-3 grey-opacity'/>
+                </div>
```
Comment (kagaramag) [inline src/components/Quiz/quiz.jsx]: Put this in a separate component

Comment (MBenedicte) [reply src/components/Quiz/quiz.jsx]: Done

File: src/assets/css/_opacity.scss (hunk 1 above)
Comment (MBenedicte) [reply src/assets/css/_opacity.scss]: resolved

File: src/components/Game/Quiz/Input/Input.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Input extends Component {
+  render() {
+    const { name, type, value } = this.props;
+    console.log('props-->', this.props);
```
Comment (sengayire) [inline src/components/Game/Quiz/Input/Input.jsx]: remove the console in you code

File: src/components/Game/Quiz/Input/Input.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Input extends Component {
```
Comment (sengayire) [inline src/components/Game/Quiz/Input/Input.jsx]: it can be better if you make this component a function based because it does not have any state

**Review by sengayire — CHANGES_REQUESTED** [review]

File: src/components/Game/Quiz/Input/Input.jsx (hunk 4 above)
Comment (MBenedicte) [reply src/components/Game/Quiz/Input/Input.jsx]: resolved

**Pull request merged**